comparisontool/static/comparisontool/js/comparison-tool.js

```diff
@@ -1868,7 +1880,7 @@ var CFPBComparisonTool = (function() {
                     });
                 });
                 request.fail(function() {
-                    // Your fail message here.
+                    // console.log( 'schoolData fetch failed')
```

Comment:
Should this be removed?

Reply:
This code is full of debugging `console.log` calls, and I plan to remove all of them when I clean up this code in another PR. :)